FE/src/components/ProfileInfo/ProfileInfo.tsx

```diff
@@ -0,0 +1,36 @@
+import {
+  Content,
+  Nickname,
+  PersonalInfoLeftDiv,
+  PersonalInfoRightDiv,
+  PointMarket,
+  ProfileImage,
+  Rank,
+  Section,
+  Topic,
+} from './ProfileInfo.styles';
+import Programmer from '/images/Programmer.png';
+
+const dummyTopic = ['등수', '포인트', '전체질문', '전체답변', '채택답변'];
+const dummyContent = ['1817', '543', '13건', '6건', '3건'];
+
+export function ProfileInfo() {
+  return (
+    <Section>
+      <PersonalInfoLeftDiv>
+        <ProfileImage src={Programmer} />
```

Comment:
프로필 이미지가 없을 경우에 빈 프로필 이미지가 보이도록 조건을 걸어주는 것이 좋을 것 같아요.
현재 백엔드에서 프로필 이미지도 함께 내려주므로 적용 가능할 것 같습니다.
```suggestion
        <ProfileImage src={userInfo.profileImage ? ProfileImage : DummyProfileImage } />
```